translations/base-de.yaml

```diff
@@ -362,23 +361,23 @@ dialogs:
         missingMods: Fehlende Mods
         newMods: Neu installierte Mods
     resourceLoadFailed:
-        title: Resources failed to load
-        demoLinkText: shapez demo on Steam
-        descWeb: "One ore more resources could not be loaded. Make sure you have a
-            stable internet connection and try again. If this still doesn't
-            work, make sure to also disable any browser extensions (including
-            adblockers).<br><br> As an alternative, you can also play the
-            <demoOnSteamLinkText>. <br><br> Error Message:"
-        descSteamDemo: "One ore more resources could not be loaded. Try restarting the
-            game - If that does not help, try reinstalling and verifying the
-            game files via Steam. <br><br> Error Message:"
+        title: Ressourcen konnten nicht geladen werden
+        demoLinkText: shapez demo-Version auf Steam
+        descWeb: "Eine oder mehrere Ressourcen konnten nicht geladen werden. Stellen Sie sicher, dass Sie eine
```

Comment:
In der deutschen Übersetzung haben wir uns für das "du" entschieden. Bis das Ganze geändert wird, bitte auch so übernehmen

Reply:
"Du" klein oder gross? Ich habe meine Grammattik in den 80ern gelernt und da war es noch gross

Reply:
Klein, so wie in der ganzen anderen Übersetzung